.github/workflows/sync-branches.yaml

```diff
@@ -30,16 +30,16 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Set up Git
-      run: |
-        git config user.name "github-actions[bot]"
-        git config user.email "github-actions[bot]@users.noreply.github.com"
-
     - name: Checkout repository
       uses: actions/checkout@v4
       with:
         fetch-depth: 0
 
+    - name: Set up Git
+      run: |
+        git config user.name "github-actions[bot]"
+        git config user.email "github-actions[bot]@users.noreply.github.com"
```

Comment:
Wouldn't adding the `--global` as is done in the other GHA help this actually? Instead of moving this down?

Reply:
True, I updated it like that. Tnx!